staging/traefik/ci/test-values.yaml

```diff
@@ -0,0 +1,16 @@
+# These test values are used for chart testing
+# DO NOT EDIT unless you know what you are doing
```

Comment:
Could we perhaps expand this comment so that it suggests how someone could go about learning what they need to learn to understand what they're doing when adding tests to this chart later?

Reply:
😬

Reply:
lol what should I take that to mean? :joy:

Reply:
I don't consider this one a blocking comment, but maybe links to the test or something could be helpful for code readers :+1:

Reply:
I agree. This warning has no value if there's no way for me to make sure I know what I'm doing.

Reply:
added comments